region/brinstar/pink.json

```diff
@@ -1937,6 +2129,70 @@
                   ],
                   "clearsObstacles": ["A"],
                   "note": "All beam weapons that can take out a Sidehopper in 5 hits or less can reliably take out the Sidehoppers damage-free."
+                },
+                {
+                  "name": "Doorway Sidehopper Kill",
+                  "notable": false,
+                  "requires": [
+                    {"doorUnlockedAtNode": 1},
+                    "canDodgeWhileShooting",
+                    {"or":[
+                      "Morph",
+                      "Spazer",
+                      "Wave",
+                      "Plasma",
+                      {"enemyKill":{
+                        "enemies": [ ["Sidehopper"] ],
+                        "explicitWeapons": [ "Missile" ]
+                      }},
+                      {"enemyDamage": {
+                        "enemy": "Sidehopper",
+                        "type": "contact",
+                        "hits": 2
+                      }},
+                      {"and":[
+                        "canTrickyJump",
```

Comment:
Does this work by walking through the hopper with iframes, just taking 1 fewer hit?